### PR TITLE
Simplify updateSchemas and allow any 'version' field

### PR DESCRIPTION
**Pull request by lundibundi**

### CODE REVIEW

File: lib/application.js [hunk 1]
```diff
@@ -372,7 +372,6 @@ class Application extends api.events.EventEmitter {
       const loadSchema = api.util.callbackify(api.metaschema.fs.load);
       loadSchema(paths, options, config, (err, schema) => {
         if (err) {
-          this.logException(err);
```
Comment (tshemsedinov) [inline lib/application.js]: Why don't log schema loading errors?

Comment (lundibundi) [reply lib/application.js]: This function may be used by the user applications to manually load and update schemas, therefore this output will only clutter the logs while user can decide on the appropriate handling himself.

The error is now logged at https://github.com/metarhia/impress/pull/1154/files#diff-5372f626ee15242f1e2c6eb31655b4faR423 where we call this function.

**Comment by nechaido** [comment]
@tshemsedinov ping.

**Comment by lundibundi** [comment]
@tshemsedinov ping.

**Comment by tshemsedinov** [comment]
Landed in https://github.com/metarhia/impress/commit/b1fa5a82c4943d62d66456e23c16681d070d7ba5

**Pull request closed without merging**